.azure-pipelines/powershell-core.yml

```diff
@@ -1,19 +1,19 @@
 variables:
-  WindowsName: windows
-  WindowsImage: windows-2019
+  WindowsName: windows 
+  WindowsAgentPoolName: pool-windows-2019
+  WindowsAgentPoolVMImage: ''
   LinuxName: linux
-  LinuxImage: ubuntu-20.04
+  LinuxAgentPoolName: pool-ubuntu-2004
+  LinuxAgentPoolVMImage: ''
   MacOSName: macOS
-  MacOSImage: macOS-11
+  MacOSAgentPoolName: 'Azure Pipelines'
+  MacOSAgentPoolVMImage: macOS-11
   TestFramework: netcoreapp3.1
   TestTarget: Test
   Configuration: Debug
-  DebugLocalBuildTasks: true
-  IsGenerateBased: $[eq(variables['system.pullRequest.targetBranch'], 'generation')]
   BuildTimeoutInMinutes: 120
   AnalysisTimeoutInMinutes: 120
   TestTimeoutInMinutes: 180
-  BuildAzPredictor: false
```

Comment:
Are Lin11-12 and 16 removed intentionally?

Reply:
- I remember the build logics are different between generation branch and other branches. They are distinguished by  IsGenerateBased. 
- BuildAzPredictor is used to skip AzPredictor build task. 
@wyunchi-ms  Could you help confirm these logics if still exist?

Reply:
This variable is still in use. Please do not remove it.